### PR TITLE
added one site to the list on web development

### PR DESCRIPTION
**Pull request by kendoriddy**
I added one more resource to Web development

### CODE REVIEW

**Comment by HareerSakhi** [comment]
@kendoriddy Just to make you aware, you need a space after your bullet point :

image
 
and also, your opening parentheses should be right next to your closing array bracket : 

image

Then everything should be ok and it can be merged.

**Comment by MRslimcoder** [comment]
@HareerSakhi amazing 🔥

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ This collection provides a list of educational resources for Software Engineers.
 - [CS50 Web](https://cs50.harvard.edu/web/2020/)- Diving more deeply into the design and implementation of web apps with Python, JavaScript, and SQL.
 - [Codementor](https://www.codementor.io/web-dev-experts)- Improve your coding skills or get help with an ongoing web development project.
 - [HTML elements PDF](https://html.com/wp-content/uploads/html-cheat-sheet.pdf) - Become a HTML geek by finding definitions, examples and references clearly with this PDF.
+-[Online Tutorials] (https://www.youtube.com/watch?v=weZFfrjF-k4&list=RDCMUCbwXnUipZsLfUckBPsC7Jog&index=2) - Find projects on HTML, CSS, and Javascript on this youtube page
```
Comment (MRslimcoder) [inline README.md:70]: Do changes suggested by @HareerSakhi

Comment (HareerSakhi) [reply README.md:70]: Thanks @MRslimcoder 😀

Comment (MRslimcoder) [reply README.md:70]: @HareerSakhi can you please share your email want to discuss something related to this project.

Comment (HareerSakhi) [reply README.md:70]: Could we please discuss it here?

Comment (MRslimcoder) [reply README.md:70]: Yeah! so keep your more focus on this repo, review more and more PR's  also send more PR's, 
If you wanr to contribute more then checkour this organization: https://github.com/devcreatives

**Comment by MRslimcoder** [comment]
@AvidCoder101 I am merging this PR, we will fix it later on.

**Comment by MRslimcoder** [comment]
@all-contributors add @kendoriddy for content.

File: README.md (hunk 1 above)
Comment (kendoriddy) [reply README.md:70]: @AvidCoder101, @MRslimcoder  done

**Pull request merged**